### DISCUSSION TITLE
Not able to use SystemMessagePromptTemplate with OpenAIConversationalAgent

### DISCUSSION THREAD

**Discussion by mahmoudsorf** [langflow-ai/langflow#768] [Q&A]
what could be the issue?

![image](https://github.com/logspace-ai/langflow/assets/141017916/cb178f3c-1757-46f3-89e7-0652f1ddf349)

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
